app/features/modelling/line-plot/index.js

```diff
@@ -0,0 +1,34 @@
+import { Chart } from 'chart.js';
+
+// Help at http://www.chartjs.org/docs/latest
+class LinePlotModel {
+  constructor(canvas, { frame, src }) {
+    this.ctx = canvas.getContext('2d');
+    console.log('Line Plot', this.ctx, frame, src);
+    this.frame = frame;
+    fetch(`${src}?=`)
+      .then(response => response.json())
+      .then((data) => {
+        console.log({
+          type: 'scatter',
+          data
+        });
+        this.lineChart = new Chart(
+          this.ctx,
+          {
+            type: 'scatter',
+            data,
+            options: {}
+          }
+        );
+      })
+      .catch((e) => {
+        console.warn(e);
+      });
+  }
+  update() {
```

Comment:
This method just returns null... is this class a work in progress and is going to be built out more later? If so, could there be a TODO comment left? If not, then does this need to be a class? Perhaps what's happening in the constructor could just be an exported function?

Reply:
the `update()`  function is required for a canvas-builder model, and since this doesn't subclass anything I needed to add in a placeholder of some kind. Is there a best practise for this?